holoviews/plotting/bokeh/util.py

```diff
@@ -177,3 +179,33 @@ def hsv_to_rgb(hsv):
     rgb = clist[order[i], np.arange(np.prod(shape))[:,None]]
 
     return rgb.reshape(shape+(3,))
+
+
+def update_plot(old, new):
```

Comment:
Seems like something that would ideally be offered by Bokeh itself (if I understand it right).